src/Template/Bake/Template/index.ctp

```diff
@@ -16,6 +16,7 @@
 <?php
 /**
   * @var \<%= $namespace %>\View\AppView $this
+  * @var \<%= $namespace %>\Model\Entity\<%= $entityClass %>[] $<%= $pluralVar %>
```

Comment:
`|\Cake\Collection\CollectionInterface` maybe, given that this is the default for baked controllers?